source_code/main_mcu/src/main.c

```diff
@@ -315,6 +316,9 @@ void main_platform_init(void)
     }
 #endif
     
+    /* Initialize ecc256 crypto engine. Uses RNG to initialize seed */
+    crypto_ecc256_init();
```

Comment:
I'm not sure how I feel about this, as this is in the context of FIDO crypto only (as you stated in the beginning of crypto.c).  
Moreover, shouldn't we merge your crypto.c with https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/LOGIC/logic_encryption.c ? 
In there are already different places containing crypto inits at user login/logout

Reply:
1) Where do you want to put the call to initialization? logic_encryption.c?
2) I can look at merging crypto.c into logic_encryption.c

Reply:
1. logic_encryption_init_context
2. nice :)